.github/workflows/node-ci.yml

```diff
@@ -28,40 +28,50 @@ jobs:
       uses: jaredly/get-changed-files@v1.0.1
       id: changed
       with:
-        format: space-delimited
-        absolute: true
+        format: json
+
+    - id: js-files
+      name: Find .js changed files
+      uses: ./.github/actions/filter-files
+      with:
+        changed-files: ${{ steps.changed.outputs.added_modified }}
+        extensions: '.js'
```

Comment:
Should we also do `.jsx` files here or in another step?

Reply:
oh yeah here too